include/pstore/core/hamt_map.hpp

```diff
@@ -457,6 +458,27 @@ namespace pstore {
             /// internal or linear node.
             void delete_node (index_pointer node, unsigned shifts);
 
+            static constexpr auto internal_nodes_per_chunk =
+                std::size_t{256} * 1024 / sizeof (internal_node);
+
```

Comment:
Again, should `256` and `1024` be declared as constant (with a meaningful name)?

Reply:
> should 256 and 1024 be declared as constant (with a meaningful name)?
 
From my POV, `internal_nodes_per_chunk` *is* that meaningful name!

Reply:
Ok, it is fine.